src/Raven.Server/RavenServerStartup.cs

```diff
@@ -232,6 +214,31 @@ public static bool IsHtmlAcceptable(HttpContext context)
             }
         }
 
+        private void LogTrafficWatch(HttpContext context, long elapsedMilliseconds, string database)
+        {
+            if (TrafficWatchManager.HasRegisteredClients)
```

Comment:
Can probably move this outside the method, so we won't even make the method call in most cases.